apps/api_web/mix.exs

```diff
@@ -60,7 +60,7 @@ defmodule ApiWeb.Mixfile do
     [
       {:phoenix, "< 1.4.15"},
       {:phoenix_html, "~> 2.11"},
-      {:ja_serializer, github: "mbta/ja_serializer", branch: "master"},
+      {:ja_serializer, github: "paulswartz/ja_serializer", branch: "mbta"},
```

Comment:
Just to clarify, I assume you're undoing this once the branch in `ja_serializer` gets merged?

Reply:
correct